dist/css/style.css

```diff
@@ -790,6 +790,7 @@ header .logo {
   margin: 5%;
   padding: 0 1rem;
   align-items: center;
+  height: 46rem;
 }
 
```

Comment:
- [ ] Please as requested by the previous code reviewer, implement the proper styling for the modal popup on mobile screens, Now when I click on the see project button of any of your project cards, I can't scroll to the top to click on the popup window close button to click on it in order to close the popup window. Kindly fix this to make your projects modal window works perfectly on a mobile screen :pray: 

- [ OPTIONAL ] Why dont you remove the horizontal overflow of your project on the desktop screen to make your project responsive :+1: